addons/website_slides/view/website_slides_embed.xml

```diff
@@ -187,22 +187,22 @@
                                 <div class="col-lg-8 col-md-8 col-sm-8 col-xs-8">
                                     <div class="col-lg-4 col-md-4 col-sm-4 col-xs-4 text-center"></div>
                                     <div class="col-lg-1 col-md-1 col-sm-1 col-xs-1 text-center">
-                                        <span id="first" title="First slide">
+                                        <span id="first" class="hide slide_nav_button" title="First slide">
```

Comment:
only add the class `o_slide_navigation_buttons` on the parent div (col-lg-8 col-md-8 col-sm-8 col-xs-8), and apply the `hide()`method if the number of page is 1.
Maybe you will have to add the `pull-right text-right` on the parent of  'download' and 'full screen' buttons to keep them on the right side.